manifests/cf-manifest/manifest/020-cf-properties.yml

```diff
@@ -495,10 +495,10 @@ properties:
       dropsonde_port: (( grab properties.metron_agent.dropsonde_incoming_port ))
 
     bbs:
-      active_key_label: keylabel1
+      active_key_label: key-2017-01
       encryption_keys:
-        - label: keylabel1
-          passphrase: "a secure passphrase"
```

Comment:
In the the commit messages, and PR description you mention that the old key is retained for now to enable migration, yet it's being removed here.